docs/ex-replace.md

```diff
@@ -82,6 +82,59 @@ Deployments:
 ...
 ```
 
+In this second example, we will override the `podman` rpm package with the version of the package used in the Fedora *updates-testing* repository. The URL of the repository can be chosen from those listed in the [Fedora Mirror List](https://mirrors.fedoraproject.org/mirrorlist?repo=updates-testing-f38&arch=x86_64). 
+```
+$ cat > /etc/yum.repos.d/updates.repo << EOF
```

Comment:
There should already be those repo configs in /etc/yum.repos.d/fedora-updates-testing.repo. They only need to be enabled. You can use `sed` to tweak the config to enable them.